game.js

```diff
@@ -36,6 +22,75 @@ const saveStates = (playerState, savedState) => {
   });
 };
 
+const createBoard = (size) => {
+const gameContainer =  document.querySelector('.game');
+gameContainer.classList.remove("hidden");
+if(game.boardSize !== "")
+gameContainer.classList.remove(game.boardSize);
+gameContainer.innerHTML = "";
+for(let i = 0; i < (size**2); i++){
+  const cell = document.createElement('div');
+  cell.classList.add(`grid-cell`);
+  cell.dataset.value = i;
+  gameContainer.appendChild(cell);
+  game.boardSize =`x${size}`
+}
+gameContainer.classList.add(game.boardSize);
+};
+
+const transposeMatrix = (matrix) => {
```

Comment:
try to move all this logic into another module (another file. e.g. getSolutionsState.js) and import if from there (i.e. 'import {getSolutions} from "./getSolutionsState.js" ')... it'll clean your file and make it more readable

Reply:
ok! ill try